### ISSUE TITLE
I am not able to download the VOD that I want can anyone help?

### ISSUE THREAD

**Issue by DivineStaRrr** [Franiac/TwitchLeecher#551]

**Comment by joshlatimer** [comment]
Same issu here, started about 20mins ago

**Comment by Tiimsoon** [comment]
Ive got the same issue. Yesterday it worked perfectly

**Comment by Ninouey** [comment]
Yea i was DLing stuff and it indeed broke just now. It doesn't fetch the links on JDownloader 2 either, so hopefully it's just a temporary Twitch issue, and not them changing their website again :V

**Comment by wolfgangam31** [comment]
Same issue here, I really hope they didn't block that API now

**Comment by MPeti1** [comment]
I noticed it maybe yesterday too, or a few days earlier, but a few hears later it was working again

**Comment by Ninouey** [comment]
And twitch had many issues this year around these time so we can wait and hope indeed ^^

**Comment by asdfg9** [comment]
as a temporary workaround to get twitch vods you can use chrome extension "HLS Downloader":
when you have it installed, open the twitch vod -> click HLS Downloader icon in chrome to see the .m3u8 link of the vod (if link isnt there, refresh the vod page) and copypaste/dragdrop the link onto jdownloader, it will grab all the available resolutions of the vod and you can DL the one you want

**Comment by MPeti1** [comment]
> 
> 
> And twitch had many issues this year around these time so we can wait and hope indeed ^^

But their website is working correctly, even the VODs can be viewed without problem

**Comment by MPeti1** [comment]
Just a note: tried youtube-dl and it wasn't working. Then updated it (`youtube-dl -U`) and now it started downloading the stream I gave it

**Comment by MPeti1** [comment]
youtube-dl has these for Twitch in the changelog:

> [twitch] Drop legacy kraken API v5 code altogether and refactor
> [twitch:vod] Switch to GraphQL for video metadata
> [twitch] Switch access token to GraphQL and refactor (ytdl-org/youtube-dl#27646)

In the mentioned pull request they remark that it's surprising that the old access token endpoint is still working. Maybe they were just in time to switch, or someone at Twitch was reading it and remembered that it's maybe time to shut it down.

**Comment by Franiac** [comment]
@MPeti1 The website works because they use their GQL API for retrieving the access token and not the old API like TL, JDownloader and youtube-dl do. The thing is, changing to that new API is nothing that can be done in 5 minutes in TL. I am pretty occupied right now and I just hope they don't drop it immediately.

**Comment by Tahsin123456** [comment]
same

**Comment by Nexxxy** [comment]
here too :(

**Comment by LeviTheRelentless** [comment]
I am also having the same error message today.

**Comment by Zurlon** [comment]
Same for me, it stops working then starts working then stops again, and so on.

**Comment by Zurlon** [comment]
@Franiac as a temporary workaround could you allow us to input our own .m3u8 links manually? So at least we can use TL to download VODs fast without having to go through VLC or JDownloader etc.

**Comment by Ralhaz** [comment]
> Every time I try to download a video this is what it says and I don't know what to do.
> 
> System.Net.WebException: The remote server returned an error: (404) Not Found.
> at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)
> at System.Net.WebClient.DownloadString(Uri address)
> at TwitchLeecher.Services.Services.TwitchService.RetrieveVodAuthInfo(String id)
> at TwitchLeecher.Gui.ViewModels.SearchResultViewVM.DownloadVideo(String id)

Same here

**Comment by athenodora** [comment]
Same issue for me :(

**Comment by EnDronist** [comment]
Same

**Comment by ScoobyDoo2021** [comment]
Hi,

The below case was closed but I didn't see what the solution was, if any.
Can someone provide an ELI5 for the 404 error mentioned in below thread:

https://github.com/Franiac/TwitchLeecher/issues/550

**Comment by IzaGz** [comment]
same problem here, cant download, any issues ? will it be fixed ? the video will expire, i need to download it please help!!!!
`System.Net.WebException: The remote server returned an error: (404) Not Found.
at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)
at System.Net.WebClient.DownloadString(Uri address)
at TwitchLeecher.Services.Services.TwitchService.RetrieveVodAuthInfo(String id)
at TwitchLeecher.Gui.ViewModels.SearchResultViewVM.DownloadVideo(String id)`

all the same!! please help! is there any alternatives to dwonload the videos!!????

**Comment by JardaH69** [comment]
Same :(
```
System.Net.WebException: The remote server returned an error: (404) Not Found.
   at System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)
   at System.Net.WebClient.DownloadString(Uri address)
   at TwitchLeecher.Services.Services.TwitchService.RetrieveVodAuthInfo(String id)
   at TwitchLeecher.Gui.ViewModels.SearchResultViewVM.DownloadVideo(String id)
```

**Comment by Franiac** [comment]
I have a potential fix that I will try to implement.

Locking due to spam

**Comment by Franiac** [comment]
TL 1.8.2 is released, should be working again 😄

**Comment by sinanpalloks** [comment]
1.8.2 WORKS! Thank you so much for fixing it so quickly. I spend my whole day trying different downloaders and now I can use youtube-dl as well but I just took the convenience of your program for granted. I'm starting to work for some streamers and when I make a little money you will definitely get a donation. Thank you have a nice day

**Comment by NICKPlayingGames** [comment]
Well, I'm getting the same problem today.

System.Net.WebException: Der Remoteserver hat einen Fehler zurückgegeben: (404) Nicht gefunden.
   bei System.Net.WebClient.DownloadDataInternal(Uri address, WebRequest& request)
   bei System.Net.WebClient.DownloadString(Uri address)
   bei TwitchLeecher.Services.Services.TwitchService.RetrieveVodAuthInfo(String id)
   bei TwitchLeecher.Gui.ViewModels.SearchResultViewVM.DownloadVideo(String id)

**Comment by MPeti1** [comment]
Could you provide an example that caused this issue for you? I started a few different downloads, and all started progressing normally

**Comment by NICKPlayingGames** [comment]
I only just downloaded the program and it happened with the first download I tried.

**Comment by NICKPlayingGames** [comment]
Like, I wanna download my stream, and this error occurs. I just installed it and changed nothing.

**Comment by MPeti1** [comment]
Which version did you get? Probably it's actually the latest, but just to be sure.
Also, try to download the stream from [February 7](https://user-images.githubusercontent.com/9215491/107700472-d26b6280-6cb7-11eb-8055-e8d97f1e1d89.png) from a channel named "Gige". If that fails, that should be some kind of a local problem, because I can download it. I think it should be enough if you limit it to the first 5 minutes, so it finishes fast.

If it fails, please include a screenshot of the error and also the logs you get.

**Comment by MPeti1** [comment]
Oh, I just noticed you have a channel. Your latest stream (Feb 7) seems to be loading without a problem for me:
![image](https://user-images.githubusercontent.com/9215491/107700688-270edd80-6cb8-11eb-8fc1-04fdf1a16c94.png)

**Comment by Franiac** [comment]
@NICKPlayingGames this looks a lot like you don't have the latest version (TL 1.8.2) installed. Please provide more information about what you are trying to download and post the download log.

**Comment by NICKPlayingGames** [comment]
Yeah, somehow the Program was on an old version without updating itself. It works fine now, sorry for bothering you all.
